.github/workflows/stable-release.yml

```diff
@@ -0,0 +1,48 @@
+name: release
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  stable:
+    uses: the-guild-org/shared-config/.github/workflows/release-stable.yml@main
+    with:
+      releaseScript: release
+      nodeVersion: 18
+    secrets:
+      githubToken: ${{ secrets.GITHUB_TOKEN }}
+      npmToken: ${{ secrets.NODE_AUTH_TOKEN }}
+
+  docker-image:
```

Comment:
@ardatan need to test this, so let me know before you do a release in Mesh, to test the entire stable pipeline

Reply:
Sure. Feel free to merge it.